### PR TITLE
Update observability tags,views,measures names

### PR DESCRIPTION
**Pull request by bogdandrutu**
Change all agent tag's TTL to TTLNoPropagation to not propagate them to the next agent/collector/backend.

This PR changes:
1) If the default views are used for receiver/exporter metrics the metric name.
2) The label keys for all the views that are using the receiver/exporter metrics.

### CODE REVIEW

File: observability/observability.go [hunk 1]
```diff
@@ -88,31 +88,31 @@ var AllViews = []*view.View{
...
 	stats.Record(ctxWithTraceReceiverName, mReceiverReceivedSpans.M(int64(receivedSpans)), mReceiverDroppedSpans.M(int64(droppedSpans)))
 }
 
-// ContextWithExporterName adds the tag "oc_exporter" and the name of the exporter as the value,
+// ContextWithExporterName adds the tag "otelsvc_exporter" and the name of the exporter as the value,
 // and returns the newly created context. For exporters that can export multiple signals it is
 // recommended to encode the signal as suffix (e.g. "oc_trace" and "oc_metrics").
 func ContextWithExporterName(ctx context.Context, exporterName string) context.Context {
-	ctx, _ = tag.New(ctx, tag.Upsert(TagKeyExporter, exporterName))
+	ctx, _ = tag.New(ctx, tag.Upsert(TagKeyExporter, exporterName, tag.WithTTL(tag.TTLNoPropagation)))
```
Comment (tigrannajaryan) [inline observability/observability.go]: @bogdandrutu I am not familiar with how tags work. Can you please clarify why this change is needed?

Comment (bogdandrutu) [reply observability/observability.go]: Updated description. Also this just limits the scope of the tags to the current binary and does not propagate them to the next hope.

File: observability/observability.go [hunk 2]
```diff
@@ -31,18 +31,18 @@ import (
 )
 
 var (
-	mReceiverReceivedSpans = stats.Int64("oc.io/receiver/received_spans", "Counts the number of spans received by the receiver", "1")
-	mReceiverDroppedSpans  = stats.Int64("oc.io/receiver/dropped_spans", "Counts the number of spans dropped by the receiver", "1")
+	mReceiverReceivedSpans = stats.Int64("otelsvc/receiver/received_spans", "Counts the number of spans received by the receiver", "1")
```
[the remark this answers is not in the document]
Comment (bogdandrutu) [reply observability/observability.go]: I tried to also put some comments in the description about this.

[the remark this answers is not in the document]
Comment (pjanotti) [reply observability/observability.go]: @tigrannajaryan yes, we do
